app/index.html

```diff
@@ -71,6 +71,8 @@
       </div>
     </div>
 
+    <img src="images/loginarrow.png" class="loginarrow visible-lg-block" alt="loginarrow" ng-if="!loggedIn">
```

Comment:
Add `width` and `height` attributes, add `style="display: none;" and then let angular make it appear using `ng-show="loggedIn"`